view/index.html

```diff
@@ -1,104 +1,174 @@
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/main.css" />
+    <link rel="stylesheet" href="../css/browse-page.css" />
+    <link rel="stylesheet" href="../css/home-page.css" />
     <title>Clacks & Thocc - Home Page</title>
-</head>
-<body>
-  <div class="nav-lip"></div>
-  <nav class="nav">
-    <a class="title-logo" href="index.html">
-      Clacks&<span>Thocc</span>
-    </a>
-    <a class="nav-logo-link" href="index.html">
-      <img class="nav-logo" src="../images/keyboard-logo.png" alt="colored keyboard logo">
-    </a>
-    <ul>
-      <li><a href="index.html">Home</a></li>
-      <li><a href="browse-page.html">Articles</a></li>
-      <li><a href="https://www.codeandtheory.com/location/manila" target="_blank">About Us</a></li>
-    </ul>
-    <img class="nav-menu-button" src="../images/burger-menu.png" alt="menu button">
-  </nav>
+  </head>
+  <body>
+    <div class="nav-lip"></div>
+    <nav class="nav">
+      <a class="title-logo" href="index.html">
+        Clacks&
+        <span>Thocc</span>
+      </a>
+      <a class="nav-logo-link" href="index.html">
+        <img
+          class="nav-logo"
+          src="../images/keyboard-logo.png"
+          alt="colored keyboard logo"
+        />
+      </a>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="browse-page.html">Articles</a></li>
+        <li>
+          <a
+            href="https://www.codeandtheory.com/location/manila"
+            target="_blank"
+          >
+            About Us
+          </a>
+        </li>
+      </ul>
+      <img
+        class="nav-menu-button"
+        src="../images/burger-menu.png"
+        alt="menu button"
+      />
+    </nav>
     <div class="grid-container">
-        <section class="categories-section">
-          <header id="main-title" class="homepage-header">
-            <h1>CLACKS & THOCC</h1>
-          </header>
-          <div class="categories-container">
-              <a class="category-container upper-categories feature-section" href="article-page.html">
-                  <h2 class="category-header">FEATURED</h2>
-              </a>
-              <div class="bottom-categories">
-                <a class="category-container recent-post-section" href="#articles-section">
-                  <h2 class="category-header">RECENT POSTS</h2>
-                </a>
-                <a class="category-container featured-product-section" href="https://www.gloriousgaming.com/products/glorious-gmmk-pro-75-barebone-black" target="_blank">
-                  <h2 class="category-header">SPOTLIGHT</h2>
-                </a>
-              </div>
-              
+      <section class="categories-section">
+        <header id="main-title" class="homepage-header">
+          <h1>CLACKS & THOCC</h1>
+        </header>
```

Comment:
We can remove the `header` here and use only the `h1` tag. `h1` is already a heading tag and the `header` tag doesn't add any value to the semantic structure here.